### PR TITLE
Bulk delete products

### PR DESCRIPTION
**Pull request by evo0705**

### CODE REVIEW

File: src/queue/Queue.js [hunk 1]
```diff
@@ -97,4 +102,7 @@ class Queue {
 
 }
 
-module.exports = Queue
+module.exports = {
```
[the remark this answers is not in the document]
Comment (evo0705) [reply src/queue/Queue.js]: Wanted to do this at first, but fail to compile. `SyntaxError: Unexpected token import`

File: src/queue/Queue.js [hunk 2]
```diff
@@ -8,6 +8,11 @@ if (RABBITMQ_BIGWIG_URL === RABBITMQ_UNSET && MODE === "production") {
   process.exit()
 }
 
+const QUEUE_NAMES = {
```
[the remark this answers is not in the document]
Comment (evo0705) [reply src/queue/Queue.js]: Yeah, understood.

File: src/api/server/services/products/batch.js [hunk 3]
```diff
@@ -87,11 +82,11 @@ class BatchUploadService {
 
       // Publish message to queue
       switch (batchAction) {
-        case this.BATCH_ACTION.CREATE_PRODUCTS:
-          await ProductBatchUploadQueue.publish(inserted.insertedId.toString())
+        case QUEUE_NAMES.BULK_PRODUCT_UPLOAD:
```
[the remark this answers is not in the document]
Comment (evo0705) [reply src/api/server/services/products/batch.js]: Yeah, I removed this piece of shit in PR26, but still, I did not remove the QUEUE_NAMES.

File: src/queue/Queue.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (evo0705) [reply src/queue/Queue.js]: import works when I change the babelrc same as jetspree-api
```
{
  "presets": [
    "es2015"
  ]
}
```
but I am not sure with that configuration, I'll check out the doc first.

[the remark this answers is not in the document]
Comment (evo0705) [reply src/queue/Queue.js]: I end up have the .babelrc like this
```
{
  "presets": [
    ["env", {
      "targets": {
        "browsers": ["last 2 versions"],
        "node": "current"
      },
      "useBuiltIns": true
    }],
    "es2017",
    "es2015",
    "react"
  ],
  "plugins": ["transform-runtime", "transform-class-properties"]
}
```
and run with `-r babel-register`

[the remark this answers is not in the document]
Comment (ravindranpandu) [reply src/queue/Queue.js]: @evo0705 guess this will solve my issue as well, nice find 👌

**Pull request merged**